thirdeye/thirdeye-pinot/src/main/resources/scripts/getGraphPnj.js

```diff
@@ -73,7 +73,7 @@ page.onLoadFinished = function(status) {
         page.render(options.output);
         phantom.exit();
     }
-  }, 500);
+  }, 5000);
 
   interval();
```

Comment:
Why we doing this here? `interval` is a number that representing the ID value of the timer that is set, it's not a function. Can we remove this? It should already self execute with `setInterval()`. If this still an issue, we will revisit with await/async.

Reply:
This script is used for PhantomJS to open an anomaly page and take a screenshot. Then it will be put in an alert email. I increased this delay here so that the script will take the screenshot after the page is fully loaded, otherwise we get an empty picture in the email.

Reply:
Yes. I understand. I'm just curious to why we adding this line `interval();` as it's not a function. The `setInterval()` should already does that.

Reply:
I guess it's because `interval` is a function pointer to `setInterval`.  Adding `interval();` actually calls the function.

Reply:
![screen shot 2018-06-22 at 10 47 38 am](https://user-images.githubusercontent.com/8840761/41791177-d74e2bc2-7609-11e8-98f4-25a4e0578b80.png)

Reply:
Okay. It's redundant then. This was written a year ago. Maybe it's time to clean it up.

Reply:
Thanks JiHao!